File: src/gestorAplicacion/general/Documento.java
package gestorAplicacion.general;

import java.time.LocalDate;

public abstract class Documento {
	protected LocalDate fechaExpedicion;

	public Documento() {
		this(LocalDate.now());
	}

	public Documento(LocalDate fecha) {
		this.fechaExpedicion = fecha;
	}

	public LocalDate getFechaExpedicion() {
		return fechaExpedicion;
	}

	protected abstract String generarIdentificador();
}

File: python/src/gestorAplicacion/general/Documento.py
from datetime import datetime

'''# Clase para crear un documento
# @author Alejandro Alvarez
# @author Alejandra Barrientos
# @author Mateo Alvarez
# @author Miguel Barrera'''

class Documento:
    def __init__(self, fecha = datetime.today()) -> None:
        self.fechaExpedicion = fecha

    def getFechaExpedicion(self):
        return self.fechaExpedicion

    def generarIdentificador(self):
        pass